src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    public abstract double getArea();

```

Comment:
```suggestion
    double getArea();
```
you dont need it here